src/components/list/list.tsx

```diff
@@ -150,17 +150,15 @@ export class List {
 
         const listItems = this.items.filter(this.isListItem);
 
-        if (!this.multiple) {
+        if (this.multiple) {
+            this.mdcList.selectedIndex = listItems
+                .filter((item: ListItem) => item.selected)
+                .map((item: ListItem) => listItems.indexOf(item));
+        } else {
             this.mdcList.selectedIndex = listItems.findIndex(
                 (item: ListItem) => item.selected
             );
-
-            return;
         }
-
-        this.mdcList.selectedIndex = listItems
-            .filter((item: ListItem) => item.selected)
-            .map((item: ListItem) => listItems.indexOf(item));
```

Comment:
This refactor might not make much sense in isolation, but when I made the change in the next commit, I wanted to make this inversion, or "un-inversion", of the `if` statement. But then it was much cleaner to just do that refactor as a separate commit first, with no changes to the functionality.